packages/sdk/src/evm/constants/chains/ChainId.ts

```diff
@@ -16,6 +16,7 @@ export enum ChainId {
   OptimismGoerli = 420,
   Arbitrum = 42161,
   ArbitrumGoerli = 421613,
+  ArbitrumSepolia = 421614,
```

Comment:
no need to add it here. this `ChainId` enum is actually deprecated. we should be using the chains package instead

Reply:
Hmm the build would fail if I don't add it, since this enum is being referenced in other places. Should we keep it for now?
![image](https://github.com/thirdweb-dev/js/assets/26052673/0e2fc145-6aa2-431e-b194-5c479cdc1908)

Reply:
Oh wait nvm I can just remove them altogether